### PR TITLE
Fix Leaflet picking when zoomed far enough out that the world is repeated

### LINKED ISSUE #1256: Feature picking in Leaflet doesn't work when zoomed out so that the map is repeated
Clicking on a feature in the repeated part of the map (either to the left or right of the "main" map (which seems to be the map in the middle) causes a selection circle to appear on the equivalent "main" map point but no feature information is shown.

### PR DESCRIPTION
**Pull request by tephenavies**
`CesiumTileLayer.pickFeatures` can receive longitude values that are not in the range [-pi, pi). This change will wrap longitude values into the range [-pi, pi) before calling `imageryProvider.pickFeatures`.

Fixes #1256.

### CODE REVIEW

File: lib/Map/CesiumTileLayer.js [hunk 1]
```diff
@@ -170,6 +170,10 @@ var CesiumTileLayer = L.TileLayer.extend({
     },
 
     pickFeatures: function(map, longitudeRadians, latitudeRadians) {
+        // Limit longitudeRadians to the range [-pi, pi)
+        function mod(x, n) { return ((x % n) + n) % n; } // Fix JS weird modulo operator
+        longitudeRadians = mod(longitudeRadians + Math.PI, 2*Math.PI) - Math.PI;
```
Comment (kring) [inline lib/Map/CesiumTileLayer.js]: Use `CesiumMath.negativePiToPi` instead.

**Comment by kring** [comment]
Also, please update CHANGES.md.

**Comment by tephenavies** [comment]
@kring Where should I put the changelong entry? In 2.0.0 or above it?

**Comment by kring** [comment]
In 2.0.0.

**Pull request merged**